fdbrpc/sim2.actor.cpp

```diff
@@ -381,8 +381,17 @@ struct Sim2Conn : IConnection, ReferenceCounted<Sim2Conn> {
 	ACTOR static Future<Void> trackLeakedConnection( Sim2Conn* self ) {
 		wait( g_simulator.onProcess( self->process ) );
 		// SOMEDAY: Make this value variable? Dependent on buggification status?
```

Comment:
This comment seems to be obsolete.

Reply:
Oups. Created PR to remove it. https://github.com/apple/foundationdb/pull/1822